options/linux/generic/sys-reboot.cpp

```diff
@@ -0,0 +1,10 @@
+#include <errno.h>
+#include <sys/reboot.h>
+#include <mlibc/sysdeps.hpp>
+
+int reboot(int what) {
+	if (!mlibc::sys_reboot) {
+		return ENOSYS;
```

Comment:
This needs to call `MLIBC_MISSING_SYSDEP();` (grep for it and compare with other stubs. Also, is the error handling (errno vs. return value) correct?